src/components/VirtualList/VirtualListFooterBottomText.tsx

```diff
@@ -0,0 +1,9 @@
+import { Trans } from '@lingui/macro';
+
+export function VirtualListFooterBottomText() {
+    return (
+        <div className="flex items-center justify-center p-6 text-base text-secondary">
+            <Trans>You&apos;ve hit rock bottom.</Trans>
```

Comment:
```suggestion
            <Trans>You&apos;ve hit rock bottom.</Trans>
```

html entities do not work in jsx, you can replace with the character instead.